apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -55,15 +56,22 @@ export class ApplicationService {
     private readonly emailService: EmailService,
   ) {}
 
-  async hasSpouseApplied(spouseNationalId: string): Promise<boolean> {
+  async hasSpouseApplied(spouseNationalId: string): Promise<IsSpouseResponse> {
```

Comment:
Should we rename this to `spouse` or `spouseInformation` or something like that?